src/main/webapp/step2-startup.js

```diff
@@ -9,10 +11,56 @@ function startup() {
     let container = document.getElementsByClassName('main-interface')[0];
     const renderer = new Step2Renderer(container);
 
-    const eventManager = new EventManager();
     renderer.render(eventManager.getEvents());
 }
 
+/**
+ * Checks the value in all radio boxes and add data to LocalStorage
+ * TODO (remusn@) Check if radio boxes are empty
+ */
+function checkRadioBoxes() {
+    const size = document.getElementsByClassName('info-pane').length;
+    for (let index = 0; index < size; ++index)  {
+        const checkbox = document.getElementsByClassName('info-pane')[index];
+        addTransportationToEvent(checkbox, index)
+    }
+}
+
+/**
+ * Update events from LocalStorage
+ * For a transportation mode with index n (from event n to event n+1)
+ * the information is added to the n-th event data
+ */
+function addTransportationToEvent(checkbox, index) {
+    const transportation = [ "Walk", "Public transportation", "Car", "Bike" ];
```

Comment:
Can you please change the strings to:
 `'WALKING', 'TRANSIT', 'DRIVING', 'BICYCLING'`, respectively?
This is how they should be added on the Maps API. Also, it should be good if you added those constants on an object like:
```
const travelModes = {
      walking: 'WALKING',
      ...
}
```
and then for example access walking by: `travelModes.walking`.

Reply:
I modified the values, but I still need them to be an array so i can access them by using index